modules/core/src/webgl/classes/texture-cube.js

```diff
@@ -13,27 +13,31 @@ const FACES = [
 
 export default class TextureCube extends Texture {
   constructor(gl, props = {}) {
+    assertWebGLContext(gl);
+
     super(gl, Object.assign({}, props, {target: GL.TEXTURE_CUBE_MAP}));
+
     this.initialize(props);
+
     Object.seal(this);
   }
 
   /* eslint-disable max-len, max-statements */
   initialize(props = {}) {
-    const {format = GL.RGBA, mipmaps = true} = props;
+    const {/* format = GL.RGBA, */ mipmaps = true} = props;
 
-    let {width = 1, height = 1, type = GL.UNSIGNED_BYTE, dataFormat} = props;
+    // let {width = 1, height = 1, type = GL.UNSIGNED_BYTE, dataFormat} = props;
 
     // Deduce width and height based on one of the faces
-    ({type, dataFormat} = this._deduceParameters({format, type, dataFormat}));
-    ({width, height} = this._deduceImageSize({
-      data: props[GL.TEXTURE_CUBE_MAP_POSITIVE_X],
-      width,
-      height
-    }));
+    // ({type, dataFormat} = this._deduceParameters({format, type, dataFormat}));
```

Comment:
Can deduction be restored?

Reply:
Well it will have to be another promise there...